backend/db/analysis.py

```diff
@@ -156,7 +156,7 @@ def counts_update(note: str, schema: str = SCHEMA, local=False, filter_temp_refr
 
 # TODO: support multiple schema
 def counts_over_time(
-    schema: str = SCHEMA, local=False, method=COUNTS_OVER_TIME_OPTIONS[0], _print=True,
+    schema: str = SCHEMA, local=False, method=COUNTS_OVER_TIME_OPTIONS[0], verbose=True,
```

Comment:
## Other misc changes
- Update: Docstrings for several test files
- Update: Lowered verbosity of test logs
- Update: misc comments / todos